api/users/serializers.py

```diff
@@ -117,6 +123,25 @@ def get_absolute_url(self, obj):
             'version': self.context['request'].parser_context['kwargs']['version']
         })
 
+    def get_node_count(self, obj):
+        return default_node_list_queryset(model_cls=Node).filter(contributor__user__id=obj.id).count()
```

Comment:
Similar to preprint counts, this is not accounting for the fact that the user making the request could be different than the target user (the user whose node counts we want).  If I am trying to see how many nodes you have, and some of your nodes are private, I shouldn't see those private nodes in the count.